app/components/agenda/side-nav.hbs

```diff
@@ -42,14 +41,30 @@
           <div class="auk-sidebar__label">
             {{#if @isOpen}}
               <span data-test-agenda-side-nav-agenda-name>
-                {{agenda.agendaName}}
+                {{#if agenda.status.isDesignAgenda}}
+                  <strong>
+                    {{t "design-agenda"}}
+                  </strong>
+                  <Auk::Pill @skin="active-future">
+                    {{agenda.name}}
+                  </Auk::Pill>
```

Comment:
Agenda.name is een `computed.alias` van `serialnumber`, beter die laatste gebruiken zodat die alias er ooit uit kan.

Legacy data heeft meestal geen `serialNumber`

Reply:
Dat is ook de reden van die computed agendaName.
Eventueel moeten we een getter maken ipv overal if statements rond te gooien (een gelijkaardige change zit ook al in een andere PR)

Reply:
Indien er geen `serialnumber` is, liever geen pill tonen, of een lege pill?

Reply:
Is niet aan mij om te beslissen. Ik vermoed dat nieuwe designs niet veel rekening word gehouden met legacy data waar allerlei zaken ontbreken.
Ik zelf zou geen lege pill tonen, gewoon "agenda". Meestal is er dan ook maar 1 agenda in onze app.